synapse/static/client/login/js/login.js

```diff
@@ -56,6 +56,7 @@ var show_login = function() {
     }
 
     if (matrixLogin.serverAcceptsSso) {
+        $("#sso_form").attr("action", "/_matrix/client/r0/login/sso/redirect");
```

Comment:
isn't this the default?

Reply:
In which point the default? This change was needed as otherwise no SSO redirect would have happened.

Reply:
The default action for #sso_form is `/_matrix/client/r0/login/sso/redirect`: https://github.com/matrix-org/synapse/blob/master/synapse/static/client/login/index.html#L22